test/service/test_manifest.py

```diff
@@ -440,14 +440,43 @@ def test_manifest_zarr(self):
                 self.assertEqual(200, response.status_code)
                 lines = response.content.decode().splitlines()
                 file_prefix = 'output="587d74b4-1075-4bbf-b96a-4d1ede0481b2/'
+                location_prefix = f'url="{config.service_endpoint()}/repository/files'
+                file_location_postfix = '"'
                 curl_files = []
+                curl_download_urls = []
+                curl_aux_download_urls = []
                 for line in lines:
                     if line.startswith(file_prefix):
-                        self.assertTrue(line.endswith('"'))
+                        self.assertTrue(line.endswith(file_location_postfix))
                         file_name = line[len(file_prefix):-1]
                         curl_files.append(file_name)
+                    elif line.startswith(location_prefix):
+                        self.assertTrue(line.endswith(file_location_postfix))
+                        download_url = line[len(location_prefix):-1]
+                        url = furl(download_url)
+                        try:
+                            url.args['drsPath']
+                        except KeyError:
+                            curl_download_urls.append(url)
+                        else:
+                            curl_aux_download_urls.append(url)
+                    else:
+                        # The curl manifest contains a combination of curl
+                        # commands, blank lines, `output` and `url` destinations.
+                        # To verify correctness we only check `output` and `url`,
+                        # everything else can be disregarded.
+                        pass
```

Comment:
> The duplicated `self.assertTrue(line.endswith('"'))` threw me for a loop here. Personally I think this if/elif would benefit from an `else: pass` and possibly a comment on why nothing needs to be checked outside of these cases. But YMMV.

Let me know how I can improve that comment.

Reply:
I think listing the unchecked cases is sufficient. Nice work.